_release/cn/pack_downloads.md

```diff
@@ -4,14 +4,15 @@ lang: cn
 ref: release
 permalink: /cn/release/pack-downloads/
 excerpt: "ServiceComb Pack Downloads"
-last_modified_at: 2020-04-06T00:50:43-55:00
+last_modified_at: 2022-05-16T00:50:43-55:00
 ---
 
 ## 发布包
 
 | 版本号           |         源码            |           发布包         |
 | ---------------------- | --------------------------------- | --------------------------------- |
-|**Apache ServiceComb Pack 0.6.0(LATEST)**|[[src]](https://apache.org/dyn/closer.cgi/servicecomb/servicecomb-pack/0.6.0/apache-servicecomb-pack-distribution-0.6.0-src.zip) [[asc]](https://www.apache.org/dist/servicecomb/servicecomb-pack/0.6.0/apache-servicecomb-pack-distribution-0.6.0-src.zip.asc) [[sha512]](https://www.apache.org/dist/servicecomb/servicecomb-pack/0.6.0/apache-servicecomb-pack-distribution-0.6.0-src.zip.sha512)|[[Binary]](https://apache.org/dyn/closer.cgi/servicecomb/servicecomb-pack/0.6.0/apache-servicecomb-pack-distribution-0.6.0-bin.zip) [[asc]](https://www.apache.org/dist/servicecomb/servicecomb-pack/0.6.0/apache-servicecomb-pack-distribution-0.6.0-bin.zip.asc) [[sha512]](https://www.apache.org/dist/servicecomb/servicecomb-pack/0.6.0/apache-servicecomb-pack-distribution-0.6.0-bin.zip.sha512)|
+|**Apache ServiceComb Pack 0.7.0(LATEST)**|[[src]](https://apache.org/dyn/closer.cgi/servicecomb/servicecomb-pack/0.7.0/apache-servicecomb-pack-distribution-0.7.0-src.zip) [[asc]](https://www.apache.org/dist/servicecomb/servicecomb-pack/0.7.0/apache-servicecomb-pack-distribution-0.7.0-src.zip.asc) [[sha512]](https://www.apache.org/dist/servicecomb/servicecomb-pack/0.7.0/apache-servicecomb-pack-distribution-0.7.0-src.zip.sha512)|[[Binary]](https://apache.org/dyn/closer.cgi/servicecomb/servicecomb-pack/0.7.0/apache-servicecomb-pack-distribution-0.7.0-bin.zip) [[asc]](https://www.apache.org/dist/servicecomb/servicecomb-pack/0.7.0/apache-servicecomb-pack-distribution-0.7.0-bin.zip.asc) [[sha512]](https://www.apache.org/dist/servicecomb/servicecomb-pack/0.7.0/apache-servicecomb-pack-distribution-0.7.0-bin.zip.sha512)|
```

Comment:
As ASF use cdn for the artifacts download,  please change the https://apache.org/dyn/closer.cgi/servicecomb/servicecomb-pack/0.7.0/apache-servicecomb-pack-distribution-0.7.0-src.zip to
https://dlcdn.apache.org/servicecomb/servicecomb-pack/0.7.0/apache-servicecomb-pack-distribution-0.7.0-src.zip

Reply:
done